packages/mixpanel/src/platform.ts

```diff
@@ -0,0 +1,18 @@
+import env from '@beam-australia/react-env';
+import { ReferralPlatform } from '@charmverse/core/prisma';
+
+const platform = env('SCOUTGAME_PLATFORM') || process.env.REACT_APP_SCOUTGAME_PLATFORM;
+
+function isPlatform(_platform: string = ''): _platform is ReferralPlatform {
```

Comment:
we really ought to just put this in packages/utils/constants or something less opinionated than mixpanel, but this is a good step in that direction